moveit_ros/planning/planning_scene_monitor/src/current_state_monitor.cpp

```diff
@@ -401,8 +408,11 @@ void CurrentStateMonitor::tfCallback()
         continue;
       }
 
+      if (auto joint_time_it = joint_time_.find(joint); joint_time_it == joint_time_.end())
+        joint_time_.emplace(joint, rclcpp::Time(0, 0, RCL_ROS_TIME));
+
       // allow update if time is more recent or if it is a static transform (time = 0)
```

Comment:
```suggestion
      // skip if time is not more recent and is not a static transform (time = 0)
```
Comments should use the same logic as the conditionals, don't make the person parsing it invert conditions.